src/calculateRentalCost.js

```diff
@@ -4,7 +4,21 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const daysCost = days * 40;
+  const discount = 20;
+  const discount2 = 50;
```

Comment:
Consider using more descriptive variable names for `discount` and `discount2`. For example, `threeDayDiscount` and `sevenDayDiscount` would make the code more readable and self-explanatory.